kubejs/client_scripts/base/jei_descriptions.js

```diff
@@ -232,6 +232,10 @@ JEIEvents.information((event) => {
                 `May be grown like Cocoa Beans on Flashing Archwood Logs.`
             ]
         },
+        {
+            items: ['emendatusenigmatica:source_gem'],
+            text: ['Obtained by placing Lapis or Amethyst in an Imbuement Chamber.']
```

Comment:
I know this message is on the Ars Source gem, but it is clearly visible in JEI too and I feel it's redundant. I'd rather leave it off, if for no other reason so I don't need to remove it again in Expert if I change this recipe (considering removing lapis as an input)

Reply:
Sure :D